examples/websockets/server.js

```diff
@@ -0,0 +1,49 @@
+// npm install sharedb connect ws
+
+var http = require("http");
+var sharedb = require("sharedb");
+var connect = require("connect");
+
+var WebSocket = require("ws");
+var Duplex = require("stream").Duplex;
+
+var server = http.createServer(connect());
+var wss = new WebSocket.Server({
+  server: server
+});
+var share = sharedb();
+
+wss.on("connection", function(ws, req) {
+  var stream = new Duplex({
+    objectMode: true
+  });
+
+  stream.headers = ws.upgradeReq.headers;
+  stream.remoteAddress = ws.upgradeReq.connection.remoteAddress;
```

Comment:
Are these (headers / remoteAddress) necessary for the correct functioning of sharedb? Would be nice to add a comment here if so.